tests/CMakeLists.txt

```diff
@@ -18,3 +18,15 @@ add_test(NAME combined-ipc-cpp COMMAND test-driver.sh combined/tst_combined_ipc.
 add_test(NAME combined-ipc-qml COMMAND test-driver.sh combined/tst_combined_ipc.qml qml combined/combined-server.qml)
 add_test(NAME combined-inprocess-cpp COMMAND test-driver.sh combined/tst_combined_inprocess.qml cpp)
 add_test(NAME combined-inprocess-qml COMMAND test-driver.sh combined/tst_combined_inprocess.qml qml)
+
+
+
+add_subdirectory(asynctests)
+add_test(NAME asynctests-local-cpp COMMAND test-driver.sh asynctests/tst_asynctests_local.qml cpp)
+add_test(NAME asynctests-local-qml COMMAND test-driver.sh asynctests/tst_asynctests_local.qml qml)
```

Comment:
We saw that QML backend implementation is not yet supported, so this should at least be commented out (and below), with a comment.